### PR TITLE
feat: Add shouldValidateGetRequestBody

### PR DESCRIPTION
**Pull request by razor-x**

### CODE REVIEW

File: packages/nextlove/src/with-route-spec/index.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export const createWithRouteSpec: CreateWithRouteSpecFunction = ((
     authMiddlewareMap = {},
     globalMiddlewares = [],
     shouldValidateResponses,
+    shouldValidateGetRequestBody = true,
```
Comment (itelo) [inline packages/nextlove/src/with-route-spec/index.ts:52]: I believe the default should be `false`

**Comment by itelo** [comment]
can you add a test?

File: packages/nextlove/src/with-route-spec/index.ts (hunk 1 above)
Comment (razor-x) [reply packages/nextlove/src/with-route-spec/index.ts:52]: The current behavior is to validate get request bodies right? So this default is chosen to keep the current behavior with the intention to make it false with a major version bump. (Please confirm I have that right.)

I'll create an issue for the next major version to collect breaking changes to include in the next major.

**Review by seveibar — APPROVED** [review]
yayyy

**Pull request merged**